packages/core/src/App/Components/Layout/Header/dtrader-v2/Utils/account-switcher-dtrader-v2-utils.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { getCurrencyName } from '@deriv/shared';
+import { Localize } from '@deriv/translations';
+import {
+    CurrencyBusdIcon,
+    CurrencyAudIcon,
+    CurrencyBchIcon,
+    CurrencyUsdtIcon,
+    CurrencyBtcIcon,
+    CurrencyEthIcon,
+    CurrencyLtcIcon,
+    CurrencyUsdcIcon,
+    CurrencyUsdIcon,
+    CurrencyEurIcon,
+    CurrencyGbpIcon,
+    CurrencyNoneIcon,
+    CurrencyDemoIcon,
+    CurrencyMultiCollateralDaiIcon,
+    CurrencyEursIcon,
+    CurrencyIdkIcon,
+    CurrencyPaxIcon,
+    CurrencyTusdIcon,
+    CurrencyUsdkIcon,
+    CurrencyXrpIcon,
+} from '@deriv/quill-icons';
+
+export const getAccountTitle = ({
+    currency,
+    loginid,
+    is_virtual,
+    show_no_currency,
+}: {
+    currency?: string;
+    loginid?: string;
+    is_virtual?: boolean;
+    show_no_currency?: boolean;
+}) => {
+    const account_type = loginid?.replace(/\d/g, '');
+
+    if (account_type === 'MF') {
```

Comment:
can we use `BrokerCodes.MF` instead of hard coding?

Reply:
Sure

Reply:
Okey, we have no such object, only type. TBrokerCodes was created based on Jurisdiction object, which have different fields from what we needed. 
Will create such object in utils files

Reply:
Also, will replace hardcoded jurisdictions with the existing object Jurisdiction👍